soccer/gameplay/standard_play.py

```diff
@@ -82,6 +83,6 @@ def score(cls):
             return float('inf')
         else:
             if (cls.is_valid(main.situationAnalysis.getSituation())):
-                return main.situationAnalysis.inSituationScore
+                return main.situationAnalysis.inSituationScore + random.random()
```

Comment:
This is planned to be temporary as it WILL give you eye cancer if you look at the scores. A more formal way to inject randomness into play selection is pending but this won't affect anything when situational analysis is turned off.

Reply:
We can shrink the randomness and force the display in the table to have less sigfigs than the randomness to stop the cancer